pages/common/rar.md

```diff
@@ -0,0 +1,27 @@
+# rar
+
+> An archive manager with compression ratio.
```

Comment:
I'm not sure this makes sense. What about the compression ratio? What about this?
```suggestion
> The RAR archiver. Supports multi-volume archives that can be optionally self-extracting.
```